map/routing_manager.cpp

```diff
@@ -247,6 +249,15 @@ drape_ptr<df::Subroute> CreateDrapeSubroute(vector<RouteSegment> const & segment
     return nullptr;
   }
 
+  if (isRuler)
+  {
+    auto const subrouteLen = segments.back().GetDistFromBeginningMerc() - baseDistance;
+    subroute->m_headFakeDistance = -kBias;
+    subroute->m_tailFakeDistance = subrouteLen + kBias;
+    subroute->m_polyline = m2::PolylineD(points);
```

Comment:
nit: std::move(points)

Reply:
Added `std::move(points)` in two places in this method.